ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -137,6 +137,11 @@ platform:
   fi
 fi
 
+for f in "${SHARED_DIR}"/*_patch_install_config.yaml;
```

Comment:
```suggestion
shopt -s nullglob
for f in "${SHARED_DIR}"/*_patch_install_config.yaml;
```

Otherwise, when no patches are available, it tries to open the literal `*_patch_install_config.yaml`.

Reply:
I'm going to do the same for baremetal ipi:

```
echo "[INFO] Looking for patches to the install-config.yaml..."

shopt -s nullglob
for f in find "${SHARED_DIR}"/*_patch_install_config.yaml;
do
  echo "[INFO] Applying patch file: $f"
  yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" "$f"
done
```

although still undecided to use find as elsewhere to be POSIX compliant.